src/main/java/frc/robot/commands/operatorcommands/TeleClimbDown.java

```diff
@@ -17,13 +17,11 @@ public TeleClimbDown(Climber climber) {
     @Override
     public void execute() {
         climber.setLeftPercent(-0.7);
```

Comment:
This should be renamed to indicate both are being set, and not just left. Even something like `setPercent()`